src/group_theory/perm/cycles.lean

```diff
@@ -144,4 +144,27 @@ def cycle_factors [fintype α] [decidable_linear_order α] (f : perm α) :
   {l : list (perm α) // l.prod = f ∧ (∀ g ∈ l, is_cycle g) ∧ l.pairwise disjoint} :=
 cycle_factors_aux (univ.sort (≤)) f (λ _ _, (mem_sort _).2 (mem_univ _))
 
+section fixed_points
+
+lemma one_lt_nonfixed_point_card_of_ne_refl [fintype α] {σ : perm α} (h : σ ≠ equiv.refl α) :
```

Comment:
You should be able to use 1 instead of `equiv.refl` I believe, here and below (imo this is slightly more readable)

Reply:
Done; thanks! I changed the names from `*_refl` to *_one` also.